MykSuite/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Infomaniak Core - Android
+  ~ Copyright (C) 2025 Infomaniak Network SA
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<resources>
+
+    <declare-styleable name="BaseMyKSuiteChipView">
+        <attr name="myKSuiteChipBackgroundColor" format="reference" />
```

Comment:
The attribute doesn't need such a precise name, it's already necessarily only applicable to `BaseMyKSuiteChipView` so you can go with something along the lines of "backgroundColor"

Reply:
That's what I did, but then it in fact doesn't compile because it's an override of an existing android attribute
so instead of juste naming it "customBackgroundColor", I went the long run

Reply:
You can avoid overriding the existing one by not redefining the format